app/views/circuitverse/index.html.erb

```diff
@@ -40,7 +40,7 @@
             <p><%= t("circuitverse.index.main_description") %></p>
             <div class="home-buttons-container justify-content-center">
               <a class="btn primary-button home-simulator-button" href='/simulator'><%= t("launch_simulator") %></a> &nbsp;
```

Comment:
Also in desktop the padding and margin og `Learn Login Design` seems little bit odd
![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/d1d6591d-0346-48ae-8383-aae740a8c03c)

Reply:
is it looking to small?.

Reply:
I was just saying keep the button like `Launch Simulator` button in terms of font size, padding, margin

Reply:
yes i understand but the problem is that in padding and font-size  when i am trying to make it as `Launch simulator button` it goes in next line i am still finding ways that can make it better

Reply:
okay, you can make both the button either as `a` tag or `button` tag, else both will act differently

Reply:
should shift ``launch simulator button`` to left so that i can add padding??
![image](https://github.com/CircuitVerse/CircuitVerse/assets/116784047/f33a7a5f-906c-4044-a3b6-855aa61b55eb)

Reply:
i tried to make anchor as a button and button as anchor button not so much difference because of the width of the container is not large